WalletWasabi.Documentation/Guides/Wasabi_Privacy_Considerations.md

```diff
@@ -0,0 +1,55 @@
+# Wasabi Wallet Privacy Considerations
+
+> _Known limitations and how to prevent de-anonymizing yourself while using Wasabi. Note that this list is not comprehensive and other limitations or weaknesses may be found in the future._
+
+### 1. Social media posts could be linked to Wasabi transactions
+
+Users must be careful not to post in a public forum about using Wasabi within minutes or hours or maybe even days of using it to mix their own coins. This could be used to infer that the user was one of the people in one of the ZeroLink rounds that happened in that time frame. Combined with other information, it could lead to the user having their online identity associated with their coins, or any of the coins that participated in the round. (This is potentially problematic if some of the coins from the round during that time frame are later used in a crime that is being investigated.)
```

Comment:
Priority of Recommendations: 
1 --> LOW
2 --> High (#1 on the list)
3 --> This is a thought, but not a recommendation. It really fits in "Don't recombine" which is the #1 
4 --> Ok - so use new fresh addresses when you receive coins in Wasabi - this (and #4) need to be at the bottom under "further conmsideration"
5 --> Move to further consideration, really not that important.
6 --> Move to further consideration, this is not clearly offering advice (which needs to be at the top) 
7 --> High (#2 on the list) 
8, 9, 10 --> Move to further considerations and low priority. The worst thing you can do is go on a blockexplorer and enter your XPUB, which might be quite damaging, but otherwise this is not a huge concern.

Reply:
these are great comments, thank you @AvivMilner